src/main/java/com/mowmaster/rprocessing/tiles/TileClayBloomery.java

```diff
@@ -23,9 +27,8 @@
     public int carboncount=0;
     public int maxcarbon=8;
     public int maxore=8;
-    public int orecount=0;
-    public int oreiron=0;
-    public int oregold=0;
+    public final int maxore = 8;
```

Comment:
Good catch, I accidentally reformatted my local code and had to select the lines, which apparently didn't work out so well :(
As you don't merge it anyways, I'll leave it like that.
Btw. I'm fine with that, take whatever you want.